python/tvm/micro/contrib/zephyr.py

```diff
@@ -204,6 +204,22 @@ def library(self, output, sources, options=None):
         )
         return tvm.micro.MicroLibrary(build_dir, [f"lib{project_name}.a"])
 
+    def _print_make_statistics(self, output):
+        print("==================")
+        output = output.splitlines()
+        lines = iter(output)
+        for line in lines:
```

Comment:
could you catch StopIteration somewhere so that in the event the output looks weird, this doesn't break the compilation flow?

Reply:
done